packages/replay-next/components/sources/log-point-panel/Capsule.tsx

```diff
@@ -65,44 +70,108 @@ export default function Capsule({
 
   const badgeStyle = getBadgeStyleVars(point.badge);
 
-  let label: ReactNode = "10k+";
-  if (!tooManyPointsToFind) {
-    if (currentHitPointIndex != null) {
-      label = `${currentHitPointIndex + 1}/${hitPoints.length}`;
-    } else {
-      label = `${hitPoints.length}`;
+  const onLabelClick = () => {
+    inputRef.current!.focus();
+  };
+
+  const onBlur = ({ currentTarget }: FocusEvent<HTMLDivElement>) => {
+    deselect(currentTarget);
+  };
+
+  const onChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const input = inputRef.current!;
+    const numChars = input.value.length;
+    const maxChars = `${hitPoints.length}`.length;
+    input.style.width = `${Math.min(numChars, maxChars)}ch`;
+  };
+
+  const onKeyDown = (event: KeyboardEvent) => {
+    switch (event.code) {
+      case "Enter":
```

Comment:
:hand: Apparently this does _not_ work with `Enter` on the numpad :(

If I go to https://w3c.github.io/uievents/tools/key-event-viewer.html and first press `Enter` in the main keyboard and then `Enter` in the numpad, it looks like the `event.code` field differs (`"Enter"` vs `"NumpadEnter"`):

![image](https://user-images.githubusercontent.com/1128784/218837044-5f564092-2a98-49f2-8ebc-5046b0fcfe2f.png)


Can we tweak this to work with both `Enter` keys?  (also any other places we're checking for the `Enter` key like this?)

Reply:
Want to file a Linear issue? I don't know how much we care to support "NumpadEnter" but if we _do_ then we have a bunch of places to check for it.

<img width="325" alt="Screen Shot 2023-02-14 at 2 26 14 PM" src="https://user-images.githubusercontent.com/29597/218837301-4866d18b-b703-4d77-a6dc-319a7ee298df.png">

Reply:
Okay. It was certainly confusing for me when I tried to type in numeric values in the hit count edit box and then hit the numpad `Enter`, and nothing happened.

Reply:
That's fair. I haven't had a numpad in so long I can't remember.